lux/core/frame.py

```diff
@@ -290,8 +292,8 @@ def __repr__(self):
     #######################################################
 
     def set_SQL_connection(self, connection, t_name):
```

Comment:
Same here, wondering if we should move this outside of `frame.py`?

Reply:
maybe you can move it to the global config and put it as a setter?

Reply:
Yeah I think it would be good to have a setter in the global config for the SQL connection. I think I'd also have to add a separate setter in frame.py for the specific table name so users can associate individual tables with individual data frames still.